commit d6e05e7a26fc512995f405b8fad8df07f1e0326f (refs/changes/95/495/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-18T00:50:12+00:00 (3 years, 1 month ago)
    
    joyent/node-triton#131 update 'make cutarelease' and 'make versioncheck' to avoid commits for cutting a release (fix 'npm publish' command usage)

diff --git a/Makefile b/Makefile
index 6951165..b74b5b4 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ cutarelease: versioncheck
 	    date=$(shell date -u "+%Y-%m-%d") && \
 	    git tag -a "$$ver" -m "version $$ver ($$date)" && \
 	    git push --tags origin && \
-	    npm publish $$name
+	    npm publish
 
 .PHONY: git-hooks
 git-hooks:
